### DISCUSSION TITLE
database_url config issue

### DISCUSSION THREAD

**Discussion by YunusMechery** [sqlpage/SQLPage#687] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/e2f7539f-10e6-463a-8231-9d8ef04be04d)
What is missing. I recently had to reset my Windows 11 and had to fresh install everything. Please help.

**Comment by lovasoa** [comment, marked as answer]
Hello Yunus!

 Have a look at https://github.com/sqlpage/SQLPage/blob/main/configuration.md#database-connection-strings

  You probably forgot to url-encode your password.
